chord_metadata_service/patients/filters.py

```diff
@@ -87,11 +92,11 @@ def filter_search(self, qs, name, value):
                                 Cast("phenopackets__biosamples__procedure__body_site", TextField()),
                                 Cast("phenopackets__biosamples__procedure__extra_properties", TextField()),
                                 Cast("phenopackets__biosamples__extra_properties", TextField()),
-                                # Biosample Variant fields
-                                "phenopackets__biosamples__variants__allele_type",
-                                Cast("phenopackets__biosamples__variants__allele", TextField()),
-                                Cast("phenopackets__biosamples__variants__zygosity", TextField()),
-                                Cast("phenopackets__biosamples__variants__extra_properties", TextField()),
+
+                                # TODO: Interpretation Variant fields
+
+                                # TODO: Interpretation Gene fields
```

Comment:
these would be good to add at least a couple of although if there's anything super nested or inside a JSON field we could skip those...